Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -183,10 +183,16 @@ public void run() {
             adSizeType = BANNER_LEADERBOARD;
         } else if (adWidthInDp >= BANNER.getWidth() && adHeightInDp >= BANNER.getHeight()) {
             adSizeType = BANNER;
-        } else {
+        } else if (adWidthInDp >= BANNER_SHORT.getWidth() && adHeightInDp >= BANNER_SHORT.getHeight()) {
             adSizeType = BANNER_SHORT;
         }
```

Comment:
@palrahul @alexander-gubov : there's one difference here is iOS return short_banner as fallback when can't mapping any vungle size, and android will return null and load failed. So what's the expected?  
In my option, I think we shouldn't return short_banner as fallback. It's not make sence for requesting size like 1dp by 1 dp.

Reply:
I also think it should be an error if banner doesn't fit, but if you are changing the logic please refer to the previous ticket in PR name

Reply:
yes, if I have to change I will create new PR for this.